### DISCUSSION TITLE
posts/how-to-create-a-react-app-with-rust-and-wasm/?giscus=3b01d0e986c1d0a8736a52efaU54KHoABZQ9eG88ao83%2F1FsthZNAncOkDTOXhLIS5GnNhLcYDH%2BXUEWEj456sGpChMIEfXco5s61CD9GaYqFJ3aoGwve6Ua9rnk6%2FK8D6fVxCLetxq7wUuPnXg%3D

### DISCUSSION THREAD

**Discussion by giscus** [tkat0/tkat0.github.io#8] [Blog Comments]
# posts/how-to-create-a-react-app-with-rust-and-wasm/?giscus=3b01d0e986c1d0a8736a52efaU54KHoABZQ9eG88ao83%2F1FsthZNAncOkDTOXhLIS5GnNhLcYDH%2BXUEWEj456sGpChMIEfXco5s61CD9GaYqFJ3aoGwve6Ua9rnk6%2FK8D6fVxCLetxq7wUuPnXg%3D

In this article, I'll introduce followings through creating simple demo application.

https://www.tkat0.dev/posts/how-to-create-a-react-app-with-rust-and-wasm/

**Comment by BayernEV** [comment]
Please remove the line "import and call" from React app. Seems to be a typing mistake?
